lms/djangoapps/courseware/views.py

```diff
@@ -193,23 +193,35 @@ def get_current_child(xmodule, min_depth=None):
 
     Returns None only if there are no children at all.
     """
+    def _get_child(children):
+        """
+        Returns either the first or last child based on the value of
+        the child argument.
```

Comment:
Nit: this is based on the value of the request_child argument, not child. I would also describe that the first child is the default if no argument is specified.